### ISSUE TITLE
clearer on/off button state

### ISSUE THREAD

**Issue by thebrainfox44** [Beep6581/RawTherapee#7167]
A bit more contrast between the modules on/off buttons states icons would be welcome.

I have a small screen and cant tell if the modules are on/off easily so making it a bit bigger, color codding it or adding more contrast could help a lot!

**Comment by Benitoite** [comment]
There is a more contrasty theme than the Default:
* Preferences > General > Theme > **TooWaBlue - Dark**

**Comment by Lawrence37** [comment]
How big is your screen and what's the resolution? Do you have pseudo HIDPI enabled or are you using the development version? The brightness of the text next to the power icons changes with the state of the tools. That may help you.

**Comment by rebio** [comment]
In the past there was a different button style for OFF position which was more different from the ON position as it is today.
I always liked the old one better as it was easier and faster to see if the module was on or off.
I think [this](https://www.tuxoche.de/wp-content/uploads/2017/02/allgemin-de.png) was the old style

**Comment by thebrainfox44** [comment]
@rebio yup this is exactly what i talk about, a clearer state. it may be less stylish but i would prefer it

**Comment by Benitoite** [comment]
The power-off icon should not be so slim and dim that users can't figure out how to enable a module, though.

"dev"
<img src="https://github.com/user-attachments/assets/fe2b11c0-e05b-4a67-adce-c19f76bc918b" width=200>

"5.4"
<img src="https://github.com/user-attachments/assets/af92f2aa-1b1d-4d66-9139-91ed3d4ac655" width=200>

**Comment by Lawrence37** [comment]
Looking at @Benitoite's screenshots, I see a bigger distinction between the on and off states with dev. @thebrainfox44, did you compare with @rebio's screenshot at the same scale? What size and resolution is your screen?

**Comment by Lawrence37** [comment]
@thebrainfox44 Can you please provide the requested information and try #7640 by downloading the artifact for you computer type from the "Checks"?
